### ISSUE TITLE
Quick open: Include symbols blocks the search based on filename

### ISSUE THREAD

**Issue by Jack-Works** [microsoft/vscode#101487]
Version: 1.47.0-insider (user setup)
Commit: b9d7ec9ff367a8e59fe4cc2abee605a55be180e8
Date: 2020-06-20T00:08:33.687Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

Steps to Reproduce:

1. Open "search.quickOpen.includeSymbols"
2. Search(Ctrl+P) in a big project (for example, typescript) by filename
3. Expected: filename comes first, then the slow symbol search joins the result later.
4. Actual: The symbol search blocks the filename search and no results appear in a long time.

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
Related https://github.com/microsoft/vscode/issues/91445

Showing symbols after files might not be an ideal solution though.

**Comment by Jack-Works** [comment]
> Related https://github.com/microsoft/vscode/issues/91445
> 
> Showing symbols after files might not be an ideal solution though. 

Hi I didn't mean the order of the search results. I mean the search providers can "gradually" add their results into the result list and don't have to wait all provider finish it's searching.

**Comment by bpasero** [comment]
Yeah understood, but that might be a frustrating experience if the user started to go through the list of files and then suddenly more results are intermixed.
